### PR TITLE
new1 commit

### PR DESCRIPTION
**Pull request by deepaksh088**
this is to commit into master

### CODE REVIEW

File: DummyTesting/src/main/java/ApiVerification/ApiVerification.java [hunk 1]
```diff
@@ -14,6 +14,8 @@ public static void responseCodeValiddation(Response response, int statusCode) {
 		System.out.println("master commit c2");
 
 		System.out.println("new branch commit c3");
+
```
Comment (deepaksh08) [inline DummyTesting/src/main/java/ApiVerification/ApiVerification.java:17]: remove the spacing and raise another request

**Review by deepaksh08 — CHANGES_REQUESTED** [review]
make space dummy comment on pr

File: DummyTesting/src/main/java/ApiVerification/ApiVerification.java (hunk 1 above)
Comment (deepaksh088) [reply DummyTesting/src/main/java/ApiVerification/ApiVerification.java:17]: spacing removed please review

**Review by deepaksh08 — APPROVED** [review]
good to merge, go ahead

**Comment by deepaksh08** [comment]
request approved and merged now

**Pull request merged**